app/views/puppetclasses/_form.html.erb

```diff
@@ -73,7 +73,7 @@
           <% @puppetclass.lookup_keys.each do |key| %>
             <li>
               <a data-toggle="tab" id="pill_<%= key.to_param %>" href="#<%= key.to_param %>" title="<%= key %>">
-                <div class="clip"><%= key %></div><span class="close pull-right">&times;</span>
+                <div class="clip"><%= key.to_s.gsub('_',' ') %></div><span class="close pull-right">&times;</span>
```

Comment:
Because smart class parameters already does this and now that whitespace isn't possible titles will have underscores. It might be easier to read titles with spaces instead of underscores.